DQMOffline/Muon/interface/MuonTiming.h

```diff
@@ -0,0 +1,96 @@
+#ifndef MuonTiming_H
```

Comment:
hi @parbol - sorry I missed this before - the include guard should have the package name to ensure no conflicts - eg
ifndef DQMOffline_Muon_MuonTiming_H